### PR TITLE
hilarymasland_solution

### PR DESCRIPTION
**Pull request by hilarymasland**
comfort- 3.5
complete- 3.5

### CODE REVIEW

File: Wendy_Bite.txt [hunk 1]
```diff
@@ -9,11 +9,19 @@ Thank You for being a friend. Travel down a road and back again. Your heart is t
 RESUME PAGE
 
 Resume
+<<<<<<< HEAD
```
Comment (jshawl) [inline Wendy_Bite.txt]: this is the sign of a merge conflict. check out https://help.github.com/articles/resolving-a-merge-conflict-from-the-command-line/ and let me know if I can help!

File: index.html [hunk 2]
```diff
@@ -31,4 +57,10 @@
   </footer>
 
   </body>
+<<<<<<< HEAD
+</div>
+</html>
+</div>
+=======
 </html>
+>>>>>>> 8ed4474a823882fb8f47c5bcc097952cc65d1f7b
```
Comment (jshawl) [inline index.html]: I'm not sure why there are so many merge conflicts in this file - let me know if you;d like to review the fork / push / pull /clone process

File: style.css [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+     }
+  #container {
+      border-style: solid;
+      border-color: #686868;
+      border-width: medium;
+      box-sizing: border-box;
+      margin-top: 60px;
+      margin-left: 60px;
+      margin-right: 60px;
+      margin-bottom: 60px;
```
Comment (jshawl) [inline style.css]: can shorten to `margin: 60px;`

File: style.css [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+      margin-bottom: 10px;
+  }
+  #education {
+    font-family: sans-serif;
+  }
+  #queens {
+    font-family:sans-serif;
+  }
+  #brooklyn {
+    font-family: sans-serif;
```
Comment (jshawl) [inline style.css]: the above three rules are identical. You can combine them, like so:

```css
#education, #queens, #brooklyn{
  font-family: sans-serif;
}
```

File: style.css [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    color: #A0A0A0
+  }
+ p {
+   text-align: left;
+   size: 16px;
+   font-style: normal;
+   font-family: georgia;
+ }
+footer {
+  text-align: center;
```
Comment (jshawl) [inline style.css]: try to be consistent w/ code indentation

File: style.css [hunk 6]
```diff
@@ -0,0 +1,116 @@
...
+.masterknitter {
+  font-weight:bold;
+  font-family: sans-serif;
+  font-size: 18px;
+}
+.senior {
+  font-weight:bold;
+  font-family: sans-serif;
+  font-size: 18px;
+}
```
Comment (jshawl) [inline style.css]: combine the above rules into one comma-separated selector

File: style.css [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+.delightful {
+  color: #A0A0A0;
+}
+.managedteam {
+  color: #A0A0A0;
+}
+.wonmultiple {
+  color: #A0A0A0;
+}
```
Comment (jshawl) [inline style.css]: same comment as above

File: wendy_bite [hunk 8]
```diff
@@ -0,0 +1 @@
+Subproject commit 8ed4474a823882fb8f47c5bcc097952cc65d1f7b
```
Comment (jshawl) [inline wendy_bite]: this indicates a git repo inside of another git repository

**Comment by jshawl** [comment]
:+1:

**Comment by hilarymasland** [comment]
Yes. I think it's because I kept trying to push the file and wasn't sure
why it wasn't working....This was BEFORE you told me that cloning and git
init were the same thing. I feel pretty good about it after yesterday
afternoon.

On Fri, Feb 12, 2016 at 2:47 PM, Jesse Shawl <notifications@github.com>
wrote:

> Closed #31 <https://github.com/ga-wdi-exercises/wendy_bite/pull/31>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ga-wdi-exercises/wendy_bite/pull/31#event-548980431>.
>



-- 


Hilary Masland
(717) 440-1950

**Pull request closed without merging**